examples/test/test_single_agent.py

```diff
@@ -36,7 +36,9 @@ def test_misalignment_single_agent(model):
     examples.misalignment.single_agent.main(model=model)
 
 
-@parametrize
+@pytest.mark.parametrize('model', [
+    ModelType.STUB,
+])
```

Comment:
Why this change is necessary?

Reply:
It was too slow with the real model. We do not check the semantics anyway. Stub will do the trick for the testing purpose.

Reply:
Sure